.github/workflows/regression-tests.yml

```diff
@@ -19,25 +30,35 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: 0xPolygon/kurtosis-cdk
-          ref: main
+          ref: "v0.2.15"
           path: kurtosis-cdk
 
-      - name: Install Kurtosis CDK tools
-        uses: ./kurtosis-cdk/.github/actions/setup-kurtosis-cdk
+      - name: Install kurtosis
+        shell: bash
+        run: |
+          echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+          sudo apt update
+          sudo apt install kurtosis-cli=1.3.0
+          kurtosis version
```

Comment:
IMO it is better to stick with the GH action (it makes it more streamlined).

Reply:
will use that action from kurtosis.
On a side note we don't use the setup action in cdk monorepo